crates/fluvio-hub-util/src/hubaccess.rs

```diff
@@ -66,21 +69,21 @@ impl HubAccess {
         let status = res.status();
         match status {
             StatusCode::Created => {
-                println!("Hubid {hubid} created!");
+                println!("hub: hubid {hubid} created");
             }
             StatusCode::Ok => {
-                println!("Hubid {hubid} already set");
+                println!("hub: hubid {hubid} is set");
             }
             StatusCode::Forbidden => {
-                let msg = format!("Hubid {hubid} already taken");
+                let msg = format!("hub: hubid {hubid} already taken");
                 return Err(HubUtilError::HubAccess(msg));
             }
             StatusCode::Unauthorized => {
-                let msg = "Authorization error, try 'fluvio cloud login'".to_string();
+                let msg = "hub :authorization error, try 'fluvio cloud login'".to_string();
```

Comment:
This one is minor, but it looks like you probably want this formatted with the colon moved over one to the left.

i.e. `hub: authorization...`